### ISSUE TITLE
Fix Solution::getCost() to be const

### ISSUE THREAD

**Issue by woodbri** [woodbri/vehicle-routing-problems#8]
These functions should be const:

```
double Solution::getCost() {
    computeCosts();    // somewhere in the code the getcost returns 0 because the cost hant been computed
    return totalCost;
}

double Solution::getDistance() {
    computeCosts();
    return totalDistance;
}
```

but are not because there is a bug in the code (see comment above) this needs to be traced down and fixed. It is preventing solverTrash Neighborhoods::getMoveSavings(const Move& m) const; from compiling. Which in turn is forcing all the callers to also be non-const. :(

**Comment by cvvergara** [comment]
pushed
fix for issue #8

**Comment by woodbri** [comment]
closing.
